plugins/steve/resource-class.ts

```diff
@@ -389,11 +432,11 @@ export default class Resource {
     throw new Error('Implement setLabel in subclass');
   }
 
-  setAnnotations(val) {
+  setAnnotations() {
```

Comment:
These will need to have the correct params. Anything that does <instance of resource-class>.setAnnotations will use this signature rather than anything in the subclass

Reply:
I don't understand. Why does it need that signature? I had deleted the variable because it was unused.